tutorials/install-distro.md

```diff
@@ -0,0 +1,72 @@
+# Distribution Specific Installation Instructions
+
+## Supported Versions
+Below is a compatiblity matrix between versions of CRI-O (y-axis) and distributions (x-axis)
+
+|      | Fedora 31+ | openSUSE | CentOS_8 | CentOS_8_Stream | CentOS_7 | Debian_Unstable | Debian_Testing | Debian 10 | Rasbian_10 | xUbuntu_20.04 | xUbuntu_19.10 | xUbuntu_19.04 | xUbuntu_18.04 |
+| ---- | ---------- | -------- | -------- | --------------- | -------- | --------------- | -------------- | --------- | ---------- | ------------- | ------------- | ------------- | ------------- |
+| 1.18 | x          | x        | x        | x               |          | x               | x              |           |            | x             |               |               |               |
```

Comment:
There are no "stable" 1.18 packages available for Ubuntu 20.04 either, see #3844